pkg/library/resources/helper.go

```diff
@@ -151,24 +149,29 @@ func ProcessResources(resources *corev1.ResourceRequirements) (*corev1.ResourceR
 		delete(result.Requests, corev1.ResourceCPU)
 	}
 
-	memLimit, hasMemLimit := result.Limits[corev1.ResourceMemory]
-	memRequest, hasMemRequest := result.Requests[corev1.ResourceMemory]
-	if hasMemLimit && hasMemRequest && memRequest.Cmp(memLimit) > 0 {
-		return result, fmt.Errorf("project clone memory request (%s) must be less than limit (%s)", memRequest.String(), memLimit.String())
-	}
-
-	cpuLimit, hasCPULimit := result.Limits[corev1.ResourceCPU]
-	cpuRequest, hasCPURequest := result.Requests[corev1.ResourceCPU]
-	if hasCPULimit && hasCPURequest && cpuRequest.Cmp(cpuLimit) > 0 {
-		return result, fmt.Errorf("project clone CPU request (%s) must be less than limit (%s)", cpuRequest.String(), cpuLimit.String())
-	}
-
 	if len(result.Limits) == 0 {
 		result.Limits = nil
 	}
 	if len(result.Requests) == 0 {
 		result.Requests = nil
 	}
 
-	return result, nil
+	return result
+}
+
+// ValidateResources validates that a corev1.ResourceRequirements is valid, i.e. that (if specified), limits are greater than requests
```

Comment:
Resource requirement requests are still valid if they are equal to the limit.
The implementation of this function follows this rule, however the documentation and error message should probably be modified.

i.e.

`// ValidateResources validates that a corev1.ResourceRequirements is valid, i.e. that (if specified), limits are greater than or equal to requests`

Reply:
Good catch, will update :+1: